docs-web/configuration/policy-resource.md

```diff
@@ -335,6 +341,77 @@ policies:
 ```
 In this example the Ingress Controller will use the configuration from the first policy reference `ingress-mtls-policy-one`, and ignores `ingress-mtls-policy-two`.
 
+### EgressMTLS
+
+The EgressMTLS policy configures upstreams authentication and certificate verification.
+
+For example, the following policy will use `egress-mtls-secret` to authenticate with the upstream application and `egress-trusted-ca-secret` to verify the certificate of the application:
+```yaml
+egressMTLS:
+  tlsSecret: egress-mtls-secret
+  trustedCertSecret: egress-trusted-ca-secret
+  verifyServer: on
```

Comment:
I wonder if it make sense to use `true` here. For the boolean fields, we use `true|false`. Even for fields that correspond to directives that use `on|off`.   I think it is a bit confusing that we use `"on"` in IngressMTLS and `on` here, so I suggest using `true` here.